tools/dev_images/additional_tools/alt/binary_replace.sh

```diff
@@ -80,12 +80,12 @@ function relocate_item() {
   local new_place="${RDIR}$(dirname ${file})"
 
   mkdir -p ${new_place}
-  cp -a ${file} ${new_place}
+  cp -a --remove-destination ${file} ${new_place}
```

Comment:
We need to rebuild dev_alt image here ad this is a big problem - all containers which builded on the DEV_ALT base should be restarted in the cluster